lib/index.ts

```diff
@@ -0,0 +1,50 @@
+import { clientInfo } from './environment'
+
+declare var PercyAgent: any;
+
+/**
+ * A function to take a Percy snapshot from a Puppeteer test or script. To use in your tests:
+ *   const { percySnapshot } = require('@percy/puppeteer')
+ *
+ *   const browser = await puppeteer.launch()
+ *   const page = await browser.newPage()
+ *   await page.goto(<your.test.url>)
+ *   await percySnapshot(page, <your snapshot name>, <maybe options>)
+ *
+ * @param page Puppeteer Page object that we are snapshotting. Required.
+ * @param name Name of the snapshot that we're taking. Required.
+ * @param options Additional options, e.g. '{widths: [768, 992, 1200]}'. Optional.
+ */
+export async function percySnapshot(page: any, name: string, options: any = {}) {
```

Comment:
There seem to be Typescript type annotations for Puppeteer (cf. https://www.npmjs.com/package/@types/puppeteer). It seems to me a little overkill at this point to add those as a dependency to this package just so that `page` here can be something shinier than `any`, but let me know if you disagree!

Reply:
I disagree on this one but only very mildly. My thinking is if we're going to use TypeScript we might as well use typings where possible. I'm not too worried about adding a type dependency package as it doesn't carry a whole lot of risk.

Reply:
Added types.